cli/command/image/list.go

```diff
@@ -75,6 +81,26 @@ func runImages(ctx context.Context, dockerCLI command.Cli, options imagesOptions
 		filters.Add("reference", options.matchName)
 	}
 
+	if options.tree {
+		if options.quiet {
+			return errors.New("--quiet is not yet supported with --tree")
+		}
```

Comment:
This is probably OK for now (i.e., producing an error); we may have to look what we want the behavior to be. I think for other cases we made the `quiet` always work; i.e. `--format .... --quiet` would continue to just print the `ID` (digest) of the image.